package com.chainingException;

public class two
{
    void twoexcept() throws ArithmeticException
    {
        One one = new One();
        one.oneexcept();
    }
}
